templates/fleetdb-crdb-deployment.yaml

```diff
@@ -0,0 +1,48 @@
+{{- if eq .Values.repo_knob.use_serverservice false}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    k8s-service: fleetdb-crdb
+  name: fleetdb-crdb
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      k8s-service: fleetdb-crdb
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        k8s-service: fleetdb-crdb
+    spec:
+      containers:
+        - name: fleetdb-crdb
+          env:
+            # https://github.com/cockroachdb/cockroach/issues/81209
+            - name: COCKROACH_RAFT_CLOSEDTS_ASSERTIONS_ENABLED
```

Comment:
idk about others, but i love it when we have links to issues explaining why we are doing something that seems odd. I dont do it all the time (And should do it more often), but i feel like its really helpful in explaining why we are doing this random weird thing